source/docs/articles/sites/deploy-log.md

```diff
@@ -0,0 +1,46 @@
+---
+title: The Pantheon Deploy Log
+description: Learn how to use the Deploy Log to manage updates, feature releases and more.
+keywords: deploy, deploy log, how to deploy, deploying
+---
+Whether you're working on a new feature or fixing up bugs, the Deploy Log helps you group a batch of commits into a single deployment. Best practice is to keep logical groups of edits together and then summarzie those groups with a single deployment message.
+
+**Examples**:
+
+- Fix for issue-12345
+- New feature: User Bio Fields
+- Update contrib modules to 2.x experimental branches
+- Verified passes user acceptance tests  
+
+## Deploy to Test
+Each site on Pantheon comes with three environments: Dev, Test, and Live. The Deploy Log found on the Test environment allows you to push code up from Dev, and pull content down from Live.

```

Comment:
`while optionally pulling the database and files down from Live` - Important to make the distinction between DB/Files and "content" as the DB has more than just content in it.